substrate/frame/offences/benchmarking/src/inner.rs

```diff
@@ -185,7 +192,7 @@ where
 	// (n nominators + one validator) * slashed + Slash Reported
 	assert_eq!(
 		System::<T>::read_events_for_pallet::<pallet_staking::Event<T>>().len(),
-		1 * (offender_count + 1) as usize + 1
+		1 * (offender_count + 1) as usize + 2
```

Comment:
What is this for?

Also, a comment would be nice: there's nothing stopping someone from reverting this.

Reply:
There is an extra event emitted by Staking (Slash Computed). Added the comment.